src/pages/publication/FilesAndLicensePanel.tsx

```diff
@@ -60,6 +63,24 @@ const FilesAndLicensePanel: FC<FilesAndLicensePanelProps> = ({ uppy, setTouchedF
     [setTouchedFields]
   );
 
+  useEffect(() => {
+    // Avoid adding duplicated file names to an existing publication,
+    // since files could have been uploaded in another session without being in uppy's current state
+    uppy.setOptions({
+      onBeforeFileAdded: (currentFile: UppyFile) => {
+        if (filesRef.current.some((file: File) => file.name === currentFile.name)) {
+          uppy.info(
+            t('files_and_license.no_duplicates', { fileName: currentFile.name }),
+            NotificationVariant.Error,
```

Comment:
burde dette være en error? kanskje bedre med info?

Reply:
hehe, ja... jeg hadde variant = "info" først, men default duration til uppy er sånn 2-3sek, som veldig magert, så satte duration til det vi bruker for error (6sek), så da ble det en mismatch, så satte bare begge til error...
men det mest logiske er kanskje `variant=NotificationVariant.Error` og `duration=autoHideDuration[NotificationVariant.Error]` da?

Reply:
ja, jeg synes det